ci/templates/appveyor.yml

```diff
@@ -22,6 +22,9 @@ install:
   - echo PyPy installed
   - pypy --version
 
+  # Upgrade virtualenv for e.g. more-itertools to be handled properly.
+  # Pin it to work around https://github.com/tox-dev/tox/issues/1389.
+  - C:\Python35\python -m pip install -U virtualenv==16.5.0
   - C:\Python35\python -m pip install tox
```

Comment:
Yay for my code \o/ ;)
Boo for requiring this - but it is good to pin anyway.
(fix has been merged already, but not released yet: https://github.com/pypa/virtualenv/pull/1400)